### PR TITLE
Fix and sound

### PR DESCRIPTION
**Pull request by dearnik1**
added sounds, visual effects and obstacle moving

### CODE REVIEW

File: Assets/Scripts/Rocket.cs [hunk 1]
```diff
@@ -63,19 +89,19 @@ private void LoadNextLevel()
         Scene scene = SceneManager.GetActiveScene();
         print(scene.name);
         int currentSceneIndex = scene.buildIndex;
-        if (currentSceneIndex == SceneManager.sceneCount)
+        if (currentSceneIndex == SceneManager.sceneCountInBuildSettings - 1)
```
Comment (olegho) [inline Assets/Scripts/Rocket.cs:92]: Why did you change sceneCount property to sceneCountInBuildSettings?

File: Assets/Scripts/Oscillator.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (period <= Mathf.Epsilon) { return; }
+        float cycles = Time.time / period;
+        if (gameObject.tag == "Delayed")
+        {
+            cycles += 0.25f;    //objectDelay; FIX THIS SOMEONE PLS
```
Comment (olegho) [inline Assets/Scripts/Oscillator.cs]: What should be fixed?

File: Assets/Scripts/Rocket.cs (hunk 1 above)
Comment (dearnik1) [reply Assets/Scripts/Rocket.cs:92]: sceneCount shows number of active scenes, which is always 1. sceneCountInBuildSettings - total number of scenes ( -1 because we need index)

File: Assets/Scripts/Oscillator.cs (hunk 2 above)
Comment (dearnik1) [reply Assets/Scripts/Oscillator.cs]: if written cycles += objectDelay;  the code wouldn't work properly, even if objectDelay = 0.25f;

**Review by olegho — APPROVED** [review]
